lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -147,8 +147,8 @@ Function Add-ConstructedState($desired_state) {
 # ------------------------------------------------------------------------------
 Function Remove-ConstructedState($initial_state) {
   Try {
-    Get-ADComputer $initial_state.name `
-    | Remove-ADObject `
+    Get-ADComputer -identity $initial_state.name -credential $credential |
```

Comment:
~~~ suggestion
Get-ADComputer -Identity $initial_state.name @extra_args |
~~~

Reply:
I've tried it and it also works